services/common/rs_server_common/utils/utils2.py

```diff
@@ -47,3 +52,28 @@ def read_response_error(response):
         detail = response.content.decode("utf-8", errors="ignore")
 
     return detail
+
+
+def filelock(func, env_var: str):
```

Comment:
We should move the initialization of the database structure in a script outside the rs-server. Thus, the python code should not create any table or whatever. It will be easier to maintain the db. I don't see the usage of a filelock when working with a postgres db. If it's a must, than something is wrong somewhere and more problems in future may appear

Reply:
We decided to do this in a next sprint